scribble.go

```diff
@@ -199,6 +199,39 @@ func (d *Driver) ReadAll(collection string) ([]string, error) {
 	return records, nil
 }
 
+// List all records from a collection; this is returned as a slice of strings because
+// there is no way of knowing what type the record is.
+func (d *Driver) List(collection string) ([]string, error) {
```

Comment:
Renaming the function breaks current usage. This would be a major change for no reason other than one semantic.

Reply:
I don't suggest changing the function name. If you have the impression that I suggest removing or replacing ReadAll(), then this is not the case. I suggest adding a new function called List(). All previous functions remain unchanged. What you can see in my PR.